config/routes.rb

```diff
@@ -4,6 +4,8 @@
   constraints format: 'html' do
     resources :categories, only: 'show', path: 'browse'
 
+    get '/book', to: redirect('/book-phone')
```

Comment:
I've changed the redirect status to 302 since [it's possible] (https://github.com/rails/rails/blob/69c7c06923249e7c95491db33933bbafa19707d3/actionpack/lib/action_dispatch/routing/redirection.rb#L179) in f6649f2 instead of doing it upstream so as not to change the behaviour of [this feature test](https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/features/guides.feature#L15).